js/controller.js

```diff
@@ -635,6 +641,41 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
         this.state.mainVideoDuration = this.state.duration;
       }
     },
+  
+    onTouching: function(params) {
+      if (this.videoVr) {
+        this.mb.publish(OO.EVENTS.TOUCHING, this.focusedElement, params);
+      }
+    },
+  
+    onTouched: function () {
+      if (this.videoVr) {
+        this.mb.publish(OO.EVENTS.TOUCHED, this.focusedElement);
+      }
+    },
+  
+    setViewingDirection: function(event, yaw, roll, pitch) {
+      this.state.viewingDirection = {yaw: yaw, roll: roll, pitch: pitch};
+    },
+  
+    recreatingUI: function (event, elementId, params, settings) {
+      console.log('call recreatingUI args', arguments);
+      if (!$('.oo-player-skin').length) {
+        this.state.mainVideoInnerWrapper.append("<div class='oo-player-skin'></div>")
+      }
+  
+      //load player with page level config param if exist
+      if (params.skin && params.skin.config) {
+        $.getJSON(params.skin.config, function(data) {
+          this.state.customSkinJSON = data;
+          this.loadConfigData(this.state.playerParam, this.state.persistentSettings, data, this.state.skinMetaData);
+        }.bind(this));
+      } else {
+        this.loadConfigData(this.state.playerParam, this.state.persistentSettings, this.state.customSkinJSON, this.state.skinMetaData);
+      }
+  
+      this.accessibilityControls = new AccessibilityControls(this); //keyboard support
```

Comment:
@jpfranco , would this conflict with our existing accessibility controls?

Reply:
Yes, it looks like this would recreate the accessibility controls, which I don't think is necessary. Since we're not calling `AccessibilityControls.cleanUp()` this will also duplicate all existing event listeners. Unless there's a good reason for this I would suggest removing it.